custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -146,13 +146,21 @@ def updates(self):
                 month,
                 sum(awc_open_count) AS awc_days_open,
                 CASE WHEN (sum(awc_open_count) > 0) THEN 1 ELSE 0 END AS awc_num_open,
-                sum(pse_conducted) as awc_days_pse_conducted
+                sum(pse_conducted) as awc_days_pse_conducted,
+                sum(open_bfast_count) as open_bfast_count,
+                sum(open_hotcooked_count) as open_hotcooked_count,
+                sum(days_thr_provided_count) as days_thr_provided_count,
+                sum(open_pse_count) as open_pse_count
```

Comment:
How is this different than `awc_days_pse_conducted` which we already have? And how can you tell this is the right one to include (given that the spec says "Number of days AWC Conducted Pre-School Education", which sounds a lot like the other one)

Reply:
I included the same which is being used in MPR report. Which is coming from different question than from where `awc_days_pse_conducted` is coming. But apparently signifies same thing from the what they say in MPR and dashboard. I used MPR one as was mentioned in spec. But, I can confirm from Nitish, which is prefered.

Reply:
Resolved from NItish. We can use the existing one. @calellowitz  I am making the change in the PR.